www/js/draw-model.js

```diff
@@ -246,6 +255,10 @@ define([
 				innerHTML: innerHTML
 			}, "statemachine-demo");
 
+			if(this._mode !== "AUTHOR"){
+				var nodeStatusClass = this.getStatus(node);
```

Comment:
I dont like the mode field which is coming from the request effecting so many things. Let's do one thing, set a field in model and use that? You can call it paranoia but those parameters are supposed to let us know and then we internally handle everything. I mean in future we will have a single point where if we change something and the mode will change and the behaviors as well. I mean it is very unlikely but do you see the point that I am trying to say here? I am open for suggestions on this one.